### PR TITLE
Created Security Policy

### PR DESCRIPTION
**Pull request by benhc123**
Created security policy for Gratipay.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+
+**How to report a security vulnerability?**  
+If you believe you’ve found a security vulnerability in one of our products or platforms please send it to us by emailing chad@zetaweb.com. Please include the following details with your report:
+
+* Description of the location and potential impact of the vulnerability;
+* A detailed description of the steps required to reproduce the vulnerability (POC scripts, screenshots, and compressed screen captures are all helpful to us); and
+* Your name/handle and a link for recognition in our Hall of Fame.
+
+If you’d like to encrypt the information, please use our PGP Key.
```
Comment (colindean) [inline SECURITY.md]: Could link to the key.

**Comment by colindean** [comment]
lgtm aside from suggestion

**Comment by rohitpaulk** [comment]
Shouldn't this be on https://gratipay.com/about/security/? Well, a reference at least. 

Here's what's on that page right now: 

> Please email security reports to [Chad Whitacre](mailto:chad@zetaweb.com) using the key below. [Thank you](https://gratipay.com/about/security/hall-of-fame).

**Comment by benhc123** [comment]
I just wanted to make sure the security policy was okay with everyone and in the documentation before putting it on the security page.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#Gratipay Responsible Disclosure Policy 
+
+We take the security of our systems seriously, and we value the security community. The responsible disclosure of security vulnerabilities helps us ensure the security and privacy of our users.
+
+**Guidelines**  
+
+We require that all researchers:
```
Comment (whit537) [inline SECURITY.md]: "Require" seems like an overreach. What are we going to do if someone doesn't follow these guidelines? I suggest we change "require" to "ask" instead.

**Comment by whit537** [comment]
Apart from toning down "require," this looks great. Let's merge it into the content already at /about/security/. If we want a SECURITY.md in the repo then let's simply have a link in there to https://gratipay.com/about/security/.

**Comment by Changaco** [comment]
Ready for review.

**Comment by Changaco** [comment]
Fixed the style problem.

**Comment by whit537** [comment]
@Changaco I'm ready to go on this once Travis is green.

**Pull request merged**